docs/excel/custom-functions-synchronous.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Synchronous custom functions
+description: Use synchronous custom functions to support evaluation and conditional format processes in Excel.
+ms.date: 12/09/2025
+ms.localizationpriority: medium
+---
+
+# Synchronous custom functions
+
+Synchronous custom functions allow evaluate and conditional format processes to run in Excel simultaneously with the custom function. Enable synchronous support when your custom function needs to run in tandem with any of the Excel processes listed in this article. Custom functions that don't support synchronous operations return an error such as `#CALC!` or `#VALUE!` when they run at the same time as these Excel processes.
```

Comment:
Is it standard in the custom functions world to use the verb "evaluate" as a noun? Can we use the noun instead? As per style guide, use "enable" not "allow".

```suggestion
Synchronous custom functions enable the evaluation and conditional format processes to run in Excel simultaneously with the custom function. Enable synchronous support when your custom function needs to run in tandem with any of the Excel processes listed in this article. Custom functions that don't support synchronous operations return an error such as `#CALC!` or `#VALUE!` when they run at the same time as these Excel processes.
```

Reply:
Great feedback, thank you!